Sources/OktaLogger/LoggerCore/OktaLoggerLogLevel.swift

```diff
@@ -41,7 +41,7 @@ extension OktaLoggerLogLevel {
     Generate log message icon depends on the log level
     */
 
-    static func logMessageIcon(level: OktaLoggerLogLevel) -> String {
+    public static func logMessageIcon(level: OktaLoggerLogLevel) -> String {
```

Comment:
Why?

Reply:
This function is being used from a different module (FirebaseLogger) and SPM complains it can't find it.